src/index.tsx

```diff
@@ -104,6 +104,7 @@ export const Hint: React.FC<IHintProps> = props => {
             ...childProps,
             onChange,
             onBlur,
+            onFocus: onChange,
```

Comment:
could you create a separate onFocus function that handles the user's childProps.onFocus(e) just like we did in the onChange function.

Reply:
done